olca-app/src/org/openlca/app/messages.properties

```diff
@@ -663,6 +663,7 @@ GlobalParameter=Global parameter
 GlobalParameterAlreadyExistsErr=A global parameter with this name already exists.
 GlobalParameters=Global parameters
 GraphNotFullyConnectedQuestion=The product system graph is not fully connected. Calculate results anyway?
+GraphicsTheme=Graphics theme
```

Comment:
also below, "Graphics theme" is too unspecific,  "Graphical editor theme" or something like this would be more understandable for the users